draft-steele-cose-hash-envelope.md

```diff
@@ -178,7 +178,7 @@ For example:
 
 # Encrypted Hashes
 
-Should we define this?
+The cose headers defined in this document SHOULD NOT be used in unprotected or protected headers associated with COSE_Encrypt, unless the sender is comfortable disclosing metadata regarding the encrypted payload.
```

Comment:
If we wish to keep the second part about metadata, we should clarify what value is being added by merging these two.
Suggest to remove the second part

```suggestion
The cose headers defined in this document SHOULD NOT be used in unprotected or protected headers associated with COSE_Encrypt.
```